components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -555,27 +555,27 @@
     "description": ""
   },
   "onboardingPanelVerifyHeader": {
-    "message": "Verify your wallet",
+    "message": "Verify",
     "description": ""
   },
   "onboardingPanelVerifySubtext": {
-    "message": "In order to withdraw the tokens you earn for viewing ads, you’ll need to verify your wallet.",
+    "message": "In order to withdraw the BAT you earn for viewing ads, you’ll need to verify Brave Rewards.",
```

Comment:
I noticed that we mix [right single quotation marks](https://unicode-table.com/en/2019) and [apostrophes](https://unicode-table.com/en/0027) pretty much everywhere. We might want to stick with one of them at some point. 🙂